### ISSUE TITLE
Preserve the order in which POI pictures are uploaded

### ISSUE THREAD

**Issue by zstadler** [IsraelHikingMap/Site#759]
## What I expect to happen
Pictures uploaded for a POI will be presented in the order in which they were uploaded

## What really happened
The picture order is reversed

## What I did that caused the issue - step by step
-  I've added multiple images for [מיצד מאגורה](https://israelhiking.osm.org.il/poi/OSM/node_3568825997?language=he) in a single upload.
- The upload order was preserved when the images were uploaded to Wikimedia Commons
- The upload order was reversed when the URLs were assigned to the `image*` tags:

tag | URL
----|-----
image | https://upload.wikimedia.org/wikipedia/commons/2/2a/Israel_Hiking_Map_%D7%9E%D7%99%D7%A6%D7%93_%D7%9E%D7%90%D7%92%D7%95%D7%A8%D7%94_3.jpeg
image1 | https://upload.wikimedia.org/wikipedia/commons/8/86/Israel_Hiking_Map_%D7%9E%D7%99%D7%A6%D7%93_%D7%9E%D7%90%D7%92%D7%95%D7%A8%D7%94_2.jpeg
image2 | https://upload.wikimedia.org/wikipedia/commons/b/b3/Israel_Hiking_Map_%D7%9E%D7%99%D7%A6%D7%93_%D7%9E%D7%90%D7%92%D7%95%D7%A8%D7%94_1.jpeg
image3 | https://upload.wikimedia.org/wikipedia/commons/e/e8/Israel_Hiking_Map_%D7%9E%D7%99%D7%A6%D7%93_%D7%9E%D7%90%D7%92%D7%95%D7%A8%D7%94.jpeg

## My environment
* Operating System and version: Win 10 home
* Web Browser and version: Firefox 56.0.2

**Comment by zstadler** [comment]
Let's focus on the user experience first.

You wrote:

> In that case I wanted the last photo that was uploaded (by the user just now) to be the fist photo that the user sees - in order to encourage uploading photos (I think it is just a bit discouraging if you upload a photo and it is not the first photo other users will now see).

I am not sure if you insist on preserving it. As a contributor, I find it unfair.

If I am the first person to upload a picture to a POI, I will find it very discouraging if another person comes, sees the existing picture/s, uploads an additional picture, and the new picture becomes the default picture.

IMO, this could encourage trolls and create tension between contributors, even when the subsequent contributor did not mean to offend the previous contributor.

To summarize, the pictures should be tagged `image`, `image1`, `image2`, ... , and presented, in the order they were loaded.

**Comment by HarelM** [comment]
Leaving the technical discussion aside, I think we should split the discussion into two parts:
1. When uploading a few images at once - in this case is stands to reason that the order which the user selected the images will be the order which they appear.
2. When a user adds an image to a place where there were images uploaded previously - for example if you uploaded an image a year ago, and I'm now uploading a new image - the new image should be the first to be seen since it's probably more accurate of how the place looks now, IMHO.
I think that this behavior is more important than protecting against trolling.

**Comment by zstadler** [comment]
We agree on
> 1. When uploading a few images at once - in this case is stands to reason that the order which the user selected the images will be the order which they appear.
which is a good start.

If you want to replace your own picture, you may even remove the old one. If you find a picture to be incorrect, that's also o to delete it. 

However, promoting your own pictures is not ethical or fair. It is like pushing yourself the head of a queue.

**Comment by HarelM** [comment]
> promoting your own pictures is not ethical or fair. It is like pushing yourself the head of a queue.

I disagree. better if we talk about it over the phone.
